Dockerfile

```diff
@@ -73,6 +73,8 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends install -yq \
         apache2 \
         libapache2-mod-xsendfile \
+        libgomp1 \
+        libgl1-mesa-glx \
```

Comment:
@manasars What is the full list of apt packages that will be installed along with these packages?

Reply:
Hi @azhavoro ,
We have changed libgl1-mesa-glx to libgl1 now, and below are the dependent packages:

Package: libgomp1
Depends: gcc-10-base (= 10-20200411-0ubuntu1), libc6 (>= 2.17)
Breaks: gcc-4.3 (<< 4.3.6-1), gcc-4.4 (<< 4.4.6-4), gcc-4.5 (<< 4.5.3-2)

Package: libgl1
Depends: libc6 (>= 2.14), libglvnd0 (= 1.3.1-1), libglx0 (= 1.3.1-1)
Filename: pool/main/libg/libglvnd/libgl1_1.3.1-1_amd64.deb